x-pack/plugins/apm/public/components/shared/kuery_bar/index.tsx

```diff
@@ -106,7 +106,7 @@ export function KueryBar(props: {
       const suggestions = (
         (await unifiedSearch.autocomplete.getQuerySuggestions({
           language: 'kuery',
-          indexPatterns: [dataView],
+          indexPatterns: [dataView as DataView],
```

Comment:
if it's a real `dataView` object probably it make sense to cast it in 
`x-pack/plugins/apm/public/hooks/use_dynamic_data_view.ts`

Reply:
I agree with you. But if I remove it, I get an error. `Type 'DataViewTitleAndFields' is missing the following properties from type 'DataView': fieldFormatMap, type, flattenHit, metaFields, and 40 more.ts(2740)`